lib/LANraragi/Plugin/Metadata/Eze.pm

```diff
@@ -42,6 +43,11 @@ sub get_tags {
 
     my $logger = get_plugin_logger();
     my $path_in_archive = is_file_in_archive( $lrr_info->{file_path}, "info.json" );
+
+    my @suffixlist = qw(.zip .cbz .rar .cbr .tar.gz .cbt .lzma .7z .cb7 .xz .pdf .epub);
+    my ($name, $path, $suffix) = fileparse($lrr_info->{file_path}, @suffixlist);
```

Comment:
It's fine to use the classic regexp here, the `file_path` variable given by the info hash will always be an archive anyway. 
```suggestion
    my ($name, $path, $suffix) = fileparse($lrr_info->{file_path}, qr/\.[^.]*/);
```